example_publisher/api/health_check.py

```diff
@@ -0,0 +1,39 @@
+from http.server import BaseHTTPRequestHandler
+import json
+import time
+from example_publisher.config import Config
+
+
+from example_publisher.publisher import Publisher
+
+
+class HTTPRequestHandler(BaseHTTPRequestHandler):
+    publisher: Publisher = None
+    config: Config = None
+
+    def __init__(self, *args, **kwargs):
+        self.test_priod_secs: int = (
+            HTTPRequestHandler.config.health_check_test_period_secs
+        )
+        self.last_successful_update: float = (
```

Comment:
Why is `self.last_successful_update` only assigned in the constructor, am I missing something?

Reply:
Refactored